res/controllers/Pioneer-DDJ-SB3-scripts.js

```diff
@@ -0,0 +1,1791 @@
+
+var PioneerDDJSB3 = {};
+//var PioneerDDJSB3 = {};
+
+///////////////////////////////////////////////////////////////
+//                       USER OPTIONS                        //
+///////////////////////////////////////////////////////////////
+
+// If true, vinyl mode will be enabled when Mixxx starts.
+PioneerDDJSB3.vinylModeOnStartup = false;
+
+// If true, pressing shift + cue will play the track in reverse and enable slip mode,
+// which can be used like a censor effect. If false, pressing shift + cue jumps to
+// the beginning of the track and stops playback.
+PioneerDDJSB3.reverseRollOnShiftCue = false;
+
+// Sets the jogwheels sensivity. 1 is default, 2 is twice as sensitive, 0.5 is half as sensitive.
+PioneerDDJSB3.jogwheelSensivity = 1.0;
+
+// Sets how much more sensitive the jogwheels get when holding shift.
+// Set to 1 to disable jogwheel sensitivity increase when holding shift.
+PioneerDDJSB3.jogwheelShiftMultiplier = 100;
+
+// If true Level-Meter shows VU-Master left & right. If false shows level of active deck.
+PioneerDDJSB3.showVumeterMaster = false;
+
+// If true VU-Level twinkle if AutoDJ is ON.
+PioneerDDJSB3.twinkleVumeterAutodjOn = true;
+
+// If true, releasing browser knob jumps forward to jumpPreviewPosition.
+PioneerDDJSB3.jumpPreviewEnabled = true;
+// Position in the track to jump to. 0 is the beginning of the track and 1 is the end.
+PioneerDDJSB3.jumpPreviewPosition = 0.5;
+
+PioneerDDJSB3.looprollIntervals = [1 / 16, 1 / 8, 1 / 4, 1 / 2, 1, 2, 4, 8];
+
+/*
+    Pioneer DDJ-SB3 mapping for Mixxx
+    Copyright (c) 2019 Dancephy LLC (javier@dancephy.com), licensed under GPL version 3 or later
+    Copyright (c) 2017 Be (be.0@gmx.com), licensed under GPL version 2 or later
+    Copyright (c) 2014-2015 various contributors, licensed under MIT license
+
+    Contributors and change log:
+    - Be (be.0@gmx.com): update effects and autoloop mode for Mixxx 2.1, fix level meter scaling,
+      remove LED flickering when pressing shift, start porting to Components
+    - Michael Stahl (DG3NEC): original DDJ-SB2 mapping for Mixxx 2.0
+    - Joan Ardiaca Jové (joan.ardiaca@gmail.com): Pioneer DDJ-SB mapping for Mixxx 2.0
+    - wingcom (wwingcomm@gmail.com): start of Pioneer DDJ-SB mapping
+      https://github.com/wingcom/Mixxx-Pioneer-DDJ-SB
+    - Hilton Rudham: Pioneer DDJ-SR mapping
+      https://github.com/hrudham/Mixxx-Pioneer-DDJ-SR
+
+    GPL license notice for current version:
```

Comment:
Unnecessary, everything in the mixxx repo is GPLv2 or later by default.

Reply:
Pushed these changes.